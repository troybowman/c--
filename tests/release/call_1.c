../real/call_1.c