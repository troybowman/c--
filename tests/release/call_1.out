a is now: 7
b is now: 8
c is now: 10
d is now: 5
e is now: 6
f is now: 7
a is now: 6
b is now: 7
c is now: 9
d is now: 4
e is now: 5
f is now: 42
a is now: 5
b is now: 6
c is now: 8
d is now: 3
e is now: 4
f is now: 210
a is now: 4
b is now: 5
c is now: 7
d is now: 2
e is now: 3
f is now: 840
a is now: 3
b is now: 4
c is now: 6
d is now: 1
e is now: 2
f is now: 2520
a is now: 2
b is now: 3
c is now: 5
d is now: 0
e is now: 1
f is now: 5040
a is now: 1
b is now: 2
c is now: 4
d is now: -1
e is now: 0
f is now: 5040
5040
